Makefile

```diff
@@ -0,0 +1,2 @@
+zip:
```

Comment:
I think it should be called "make package" instead, since it isn't actually creating a `.zip`-file.